tiny_gltf.h

```diff
@@ -4912,14 +4892,13 @@ static void SerializeGltfBufferView(BufferView &bufferView, json &o) {
 }
 
 static void SerializeGltfImage(Image &image, json &o) {
- 	// if uri empty, the mimeType and bufferview should be set
```

Comment:
This is just the result of running `clang-format` on the file. apparently this was forgotten after making these changes.